src/sentry/uptime/issue_platform.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import annotations
+
+import uuid
+from datetime import datetime, timezone
+
+from django.conf import settings
+from sentry_kafka_schemas.schema_types.uptime_results_v1 import CheckResult
+
+from sentry.issues.grouptype import UptimeDomainCheckFailure
+from sentry.issues.issue_occurrence import IssueEvidence, IssueOccurrence
+from sentry.issues.producer import PayloadType, produce_occurrence_to_kafka
+
+
+def create_issue_platform_occurrence(result: CheckResult):
+    occurrence = build_occurrence_from_result(result)
+    produce_occurrence_to_kafka(
+        payload_type=PayloadType.OCCURRENCE,
+        occurrence=occurrence,
+        event_data=build_event_data_for_occurrence(result, occurrence),
+    )
+
+
+def build_occurrence_from_result(result: CheckResult) -> IssueOccurrence:
+    status_reason = result["status_reason"]
+    assert status_reason
+    failure_reason = f'{status_reason["type"]} - {status_reason["description"]}'
+    evidence_display = [
+        IssueEvidence(
+            name="Failure reason",
+            value=failure_reason,
+            important=True,
+        ),
+        IssueEvidence(
+            name="Duration",
+            value=str(result["duration_ms"]),
+            important=False,
+        ),
+    ]
+    request_info = result["request_info"]
+    if request_info:
+        evidence_display.append(
+            IssueEvidence(
+                name="Method",
+                value=request_info["request_type"],
+                important=False,
+            )
+        )
+        evidence_display.append(
+            IssueEvidence(
+                name="Status Code",
+                value=str(request_info["http_status_code"]),
+                important=False,
+            ),
+        )
+
+    return IssueOccurrence(
+        id=uuid.uuid4().hex,
+        resource_id=None,
+        project_id=settings.UPTIME_POC_PROJECT_ID,  # TODO: Get this from the subscription or subscription like thing
+        event_id=uuid.uuid4().hex,
+        fingerprint=[
+            result["subscription_id"]
```

Comment:
This is going to be different every time. idk if we should finger print on this yet? otherwise new issue for every failure no?

Reply:
Discussed on slack and we'll fix in the rust code instead, so it's easier to change. Also safe to merge now since we won't create issue platform issues for this type yet